driver/razercommon.h

```diff
@@ -52,6 +52,8 @@ do { \
 #define RED_PROFILE_LED   0x0C
 #define GREEN_PROFILE_LED 0x0D
 #define BLUE_PROFILE_LED  0x0E
+#define RIGHT_LED         0x10
```

Comment:
Please call it `RIGHT_SIDE_LED` and `LEFT_SIDE_LED`

Reply:
Sure thing!